### DISCUSSION TITLE
Is it possible to use RSS-to-Telegram-Bot inside a private channel?

### DISCUSSION THREAD

**Discussion by mattiaudisio** [Rongronggg9/RSS-to-Telegram-Bot#417] [Q&A] [answered]
Hello, everyone,
probably someone has already asked this question (or it is in the guide and I didn't notice it), but is it possible to use the bot even within a Private Telegram channel?
And if yes, what do I have to do to do it?
Thank you very much to those who will answer

**Comment by Rongronggg9** [comment, marked as answer]
![](https://github.com/Rongronggg9/RSS-to-Telegram-Bot/assets/15956627/04393fc7-5720-4b99-85e6-0bb91efb94d6)
